app/views/events/_applicants_overview.html.erb

```diff
@@ -63,51 +61,25 @@
       <%= link_to "Alle drucken",
           events_path, :class => 'btn btn-default'%>
     </div>
-    <div class="btn-group pull-right" role="group">
-      <% if (!@event.applications_classified?) %>
-          <div class="tooltip-wrapper disabled has-tooltip" data-toggle="tooltip" title="<%= t '.unclassified_applications_left' %>">
-            <%= button_tag t('.sending_acceptances'), type: 'button', class: 'send-emails-button btn btn-default', disabled: true %>
-          </div>
-          <div class="tooltip-wrapper disabled has-tooltip" data-toggle="tooltip" title="<%= t '.unclassified_applications_left' %>">
-            <%= button_tag t('.sending_rejections'), type: 'button', class: 'send-emails-button btn btn-default', disabled: true %>
-          </div>
-      <% elsif (@event.compute_free_places < 0)%>
-          <div class="tooltip-wrapper disabled has-tooltip" data-toggle="tooltip" title="<%= t '.maximum_number_of_participants_exeeded' %>">
-            <%= button_tag t('.sending_acceptances'), type: 'button', class: 'send-emails-button btn btn-default', disabled: true %>
-          </div>
-          <div class="tooltip-wrapper disabled has-tooltip" data-toggle="tooltip" title="<%= t '.maximum_number_of_participants_exeeded' %>">
-            <%= button_tag t('.sending_rejections'), type: 'button', class: 'send-emails-button btn btn-default', disabled: true %>
-          </div>
-      <% else %>
-          <%= button_tag t('.sending_acceptances'), type: 'button', class: 'btn btn-default', data: {toggle: 'modal', target: '#send-emails-modal', title: t('.sending_acceptances'), list: @event.email_adresses_of_accepted_applicants} %>
-          <%= button_tag t('.sending_rejections'), type: 'button', class: 'btn btn-default', data: {toggle: 'modal', target: '#send-emails-modal', title: t('.sending_rejections'), list: @event.email_adresses_of_rejected_applicants} %>
-      <% end %>
-      <%= link_to  t('events.badges.print_button_label'),
-                   event_badges_path(@event), :class => 'btn btn-default'%>
-    </div>
-  </div>
-
-</div>
 
-<!-- Bootstrap Modal for sending emails -->
-<div id="send-emails-modal" class="modal fade" role="dialog">
-  <div class="modal-dialog">
-
-    <!-- Modal content-->
-    <div class="modal-content">
-      <div class="modal-header">
-        <button type="button" class="close" data-dismiss="modal">&times;</button>
-        <h4 class="modal-title"></h4>
+    <% if not @event.applications_classified? %>
+      <div class="btn-group pull-right tooltip-wrapper has-tooltip" role="group" data-toggle="tooltip" title="<%= t '.unclassified_applications_left'%>">
```

Comment:
If you wanted to, you could make singular/plural of "Bewerbung(en)" dependent to how many applications this event has.